src/content/zh-tw/updates/2017/10/remove-shadow-piercing.md

```diff
@@ -0,0 +1,42 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: 向shadow-piercing CSS選擇器說再見。
+
+{# wf_updated_on: 2017-10-24 #}
+{# wf_published_on: 2017-10-24 #}
+{# wf_tags: webcomponents,shadowdom,style,css,deprecations,removals #}
+{# wf_featured_image: /web/updates/images/generic/styles.png #}
+{# wf_featured_snippet: 向shadow-piercing CSS選擇器說再見。 #}
+{# wf_blink_components: Blink>DOM #}
+
+# Chrome 63將刪除::shadow 和 /deep/ {: .page-title }
+
+{% include "web/_shared/contributors/robdodson.html" %}
+
+Translated by
```

Comment:
Per our [style guide](https://developers.google.com/web/resources/translations#credit), please put the translated by at the bottom of the article.